src/lib/components/DashPlayer.react.js

```diff
@@ -141,17 +142,23 @@ export default class DashPlayer extends Component {
                 height={height}
                 style={style}
                 playsinline={playsinline}
+                class={className}
             />
         );
     }
 }
 
 DashPlayer.propTypes = {
     /**
-     * The ID used to identify this compnent in Dash callbacks
+     * The ID used to identify this component in Dash callbacks
      */
     id: PropTypes.string,
 
+    /**
+     * The CSS class used to identify this component in Dash callbacks
```

Comment:
@alexshoe you commented on the readme but this is the key place to get the description correct

Reply:
Thanks for catching this! Will fix in #48